llvm/test/MC/AMDGPU/gfx12_asm_sop1.s

```diff
@@ -684,6 +684,51 @@ s_rndne_f16 s5, 0xfe0b
 s_rndne_f16 s5, 0x3456
 // GFX12: encoding: [0xff,0x6e,0x85,0xbe,0x56,0x34,0x00,0x00]
 
+s_barrier_signal -2
```

Comment:
Missing `s_get_barrier_state` tests in this file?

Reply:
Thanks !

Reply:
Patch: https://github.com/llvm/llvm-project/pull/75575